### PR TITLE
Add shift_genomic and shift_3prime to web VEP

### PR DESCRIPTION
**Pull request by aparton**

### CODE REVIEW

**Comment by dglemos** [comment]
It looks alright. With the new option, you also need to update the image in info/docs/tools/vep/online/input.html#advanced

File: tools/modules/EnsEMBL/Web/Component/Tools/VEP/InputForm.pm
Comment (helensch) [inline tools/modules/EnsEMBL/Web/Component/Tools/VEP/InputForm.pm:787]: Suggest single quotes around shift_3prime

**Comment by imransl** [comment]
I can merge this once it has been approved

**Review by dglemos — APPROVED** [review]

**Pull request merged**
